### DISCUSSION TITLE
[Help]: I can't build project because it can't find module in dependency file

### DISCUSSION THREAD

**Discussion by tummappht** [davidjerleke/embla-carousel#757] [Help] [answered]
### Summary

This is the error i get when i run 'npm run build'

```

node_modules/embla-carousel-react/esm/components/useEmblaCarousel.d.ts:9:39 - error TS2307: Cannot find module 'embla-carousel/components/Options' or its corresponding type declarations.

9     var globalOptions: Partial<import("embla-carousel/components/Options").OptionsType> | undefined;
                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in node_modules/embla-carousel-react/esm/components/useEmblaCarousel.d.ts:9

```

### If applicable, which variants of Embla Carousel are relevant to this question?

- [ ] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment, marked as answer]
@tummappht thanks for your question. Is it related to this maybe:

- https://github.com/davidjerleke/embla-carousel/discussions/726#discussioncomment-8331396

**Reply by tummappht** [reply]
thank you. it's working for me.

**Comment by davidjerleke** [comment]
- Duplicate of #726.
